src/data/finset/noncomm_prod.lean

```diff
@@ -163,6 +165,26 @@ begin
         simp [hy] } } }
 end
 
+@[to_additive]
+lemma noncomm_prod_mem_submonoid (s : multiset α)
```

Comment:
I think this should probably be called `submonoid.multiset_noncomm_prod_mem` to match `submonoid.list_prod_mem`.